test/EndToEndTests/Services/ODataDefaultService/DefaultWCFSvcDataSource.cs

```diff
@@ -1205,15 +1206,15 @@ protected override IEdmModel CreateModel()
             return DefaultInMemoryModel.CreateODataServiceModel("Microsoft.Test.OData.Services.ODataWCFService");
         }
 
-        protected override void ConfigureContainer(IContainerBuilder builder)
+        protected override void ConfigureContainer(IServiceCollection services)
         {
-            base.ConfigureContainer(builder);
-            builder.AddServicePrototype(new ODataSimplifiedOptions()
+            base.ConfigureContainer(services);
+            services.AddSingleton(sp => new ServicePrototype<ODataSimplifiedOptions>(new ODataSimplifiedOptions()

```

Comment:
Was this registered as singleton in the old version? It's not immediately clear from the diff.